qutebrowser/mainwindow/tabbedbrowser.py

```diff
@@ -689,6 +689,7 @@ def on_load_finished(self, tab, ok):
         self._update_tab_title(idx)
         if idx == self.currentIndex():
             self._update_window_title()
+            tab._handle_auto_insert_mode(ok)
```

Comment:
Since you're accessing this from the outside, `_handle_auto_insert_mode` should be renamed to `handle_auto_insert_mode` as it can't be "private" anymore.